src/components/Wizard/CreateVmWizard/CreateVmWizard.js

```diff
@@ -199,46 +219,56 @@ export class CreateVmWizard extends React.Component {
   wizardStepsNewVM = [
     {
       title: STEP_BASIC_SETTINGS,
-      render: () => (
-        <BasicSettingsTab
-          key="1"
-          namespaces={this.props.namespaces}
-          selectedNamespace={this.props.selectedNamespace}
-          templates={this.props.templates}
-          basicSettings={this.state.stepData[BASIC_SETTINGS_TAB_IDX].value}
-          onChange={this.onStepDataChanged}
-        />
-      ),
+      render: () => {
```

Comment:
every loaded object is separate request. When namespaces and templates are loaded, we can show the first step and we do not have to wait for other requests to finish (storageclasses for storage tab etc). In slow environment, it can make a difference.